src/test/java/io/jenkins/plugins/opentelemetry/BaseIntegrationTest.java

```diff
@@ -168,7 +172,7 @@ protected void dumpMetrics(Map<String, MetricData> exportedMetrics) {
             System.out.println(tree);
         }
 
-        return trees.get(0);
+        return trees.get(index);
```

Comment:
When running the same job a few times, this tree always returns the first build.